testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -652,7 +652,28 @@ def forward(self, x, y):
 input_0 = Variable(torch.randn(1, 3, 8, 6))
 input_1 = Variable(torch.randn(1, 3, 4, 3))
 model = DynamicResize()
-save_data_and_model_multy_inputs("dynamic_resize", model, input_0, input_1, version=11)
+save_data_and_model_multy_inputs("dynamic_resize_9", model, input_0, input_1, version=9)
+save_data_and_model_multy_inputs("dynamic_resize_10", model, input_0, input_1, version=10)
+save_data_and_model_multy_inputs("dynamic_resize_11", model, input_0, input_1, version=11)
+save_data_and_model_multy_inputs("dynamic_resize_12", model, input_0, input_1, version=12)
+
+class DynamicResizeScale(nn.Module):
+    def __init__(self):
+        super(DynamicResizeScale, self).__init__()
+
```

Comment:
Do we really need a constructor for this model?

Reply:
Well 100% of all models are defined with a constructor.

Reply:
Why not?
```python3
class DynamicResize(nn.Module):
    def forward(self, x, y):
        up = nn.Upsample(scale_factor=(0.5, 0.5), mode='bilinear')
        return up(x) + y
```

Reply:
done